lib/app_router.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import 'domain/repositories/user_repository.dart';
+import 'presentation/Form/bloc/myform_bloc.dart';
+import 'presentation/Form/empty_form_screen.dart';
+import 'presentation/Form/filled_form_screen.dart';
+import 'presentation/Home/bloc/home_bloc.dart';
+import 'presentation/Home/home_screen.dart';
+
+class AppRouter {
+  Route<dynamic> generateRoute(RouteSettings settings) {
+    switch (settings.name) {
+      case HomeScreen.id:
```

Comment:
Instead of creating routes in HomeScreen etc. create static const string routes here in AppRouter class only and use them.